storage/feed.go

```diff
@@ -79,6 +79,34 @@ type StartStoreEvent struct {
 	StoreID proto.StoreID
 }
 
+// StoreStatusEvent contains the current descriptor for the given store.
+//
+// Because the descriptor contains information that cannot currently be computed
+// from other events, this event should be periodically broadcast by the store
+// independently of other operations.
+type StoreStatusEvent struct {
+	StoreID proto.StoreID
+	Desc    *proto.StoreDescriptor
```

Comment:
The Desc can actually change - for some reason, it includes a store Capacity, which is recalculated every time Store.Descriptor() is called.